### PR TITLE
Feature/print auto rounds 

### PR DESCRIPTION
**Pull request by rjs68**
-Changed code so information is printed for all rounds - even when the human has been eliminated
-Separated out the printing of human info, such as number of cards and viewing their card, so that these only print if the human is alive
-Made use of IS_TEST_MODE attribute to speed up auto complete printing - I think we should rename this to something like AUTOCOMPLETE to be more intuitive!
-Changed print speeds to provide better user experience

### CODE REVIEW

**Review by hughwin — APPROVED** [review]

File: src/main/java/com/toptrumps/cli/CommandLineController.java [hunk 1]
```diff
@@ -158,12 +162,8 @@ private void startNewGame(int numberOfOpponents) {
...
         gameEngine.persistGameState(gameState);
 
-        if (!isHumanAlive) {
-            // Trigger automatic completion notice
-            view.showAutomaticCompletion();
-        }
-
-        // Invoke the lifecycle method
+        // Invoke the lifecycle method with correct printing speed
+        IS_TEST_MODE = false;
```
Comment (BelfDev) [inline src/main/java/com/toptrumps/cli/CommandLineController.java]: IS_TEST_MODE is a constant!

**Review by BelfDev — CHANGES_REQUESTED** [review]
IS_TEST_MODE is supposed to be a **constant** flag. If you really need a new flag, please rename it.

File: src/main/java/com/toptrumps/cli/CommandLineController.java (hunk 1 above)
Comment (rjs68) [reply src/main/java/com/toptrumps/cli/CommandLineController.java]: I've changed this now!

**Review by BelfDev — APPROVED** [review]
Awesome! 👍

**Pull request merged**